admin/controllers/attendance.js

```diff
@@ -258,7 +259,7 @@ controller.getDeclarationForm = async (req, res, next) => {
     if (submitted) {
       return res.redirect('/attendance/submitted')
     }
-    hdfEligibility = await headteacherDeclarationService.getEligibilityForSchool(req.user.School)
+    hdfEligibility = await headteacherDeclarationService.getEligibilityForSchool(req.user.schoolId, checkWindowData.checkEndDate, req.user.timezone)
```

Comment:
Compared with the calls at @196 and @282, this one sends `.schoolId` and the others send `.School`- can we make them consistent? 🙂

Reply:
@danwilliams thanks Dan! that is now consistent across all calls.